notebooks/models/Evaluate azure text analytics.ipynb

```diff
@@ -0,0 +1,457 @@
+     "output_type": "stream",
+     "text": [
+      "saving experiment data to experiment_20221125-162355.json\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Evaluating Azure Text Analytics.\")\n",
+    "\n",
+    "experiment = get_experiment_tracker()\n",
+    "\n",
+    "# Mapping from dataset Entities to Text Analytics Entities. \n",
+    "# All supported PII entity categories in Text Analytics are listed in this link: https://learn.microsoft.com/en-us/azure/cognitive-services/language-service/personally-identifiable-information/concepts/conversations-entity-categories\n",
+    "i2b2_entities_to_text_analytics =  {\"PERSON\":\"Person\",\n",
+    "                                \"STREET_ADDRESS\":\"Address\",\n",
+    "                                \"GPE\": \"O\",\n",
+    "                                \"PHONE_NUMBER\":\"PhoneNumber\",\n",
+    "                                \"ORGANIZATION\":\"Organization\",\n",
+    "                                \"DATE_TIME\": \"DateTime\",\n",
+    "                                \"TITLE\":\"O\",\n",
+    "                                \"CREDIT_CARD\":\"CreditCardNumber\",\n",
+    "                                \"US_SSN\":\"USSocialSecurityNumber\",\n",
+    "                                \"AGE\": \"Age\",\n",
+    "                                \"NRP\":\"O\",\n",
+    "                                \"ZIP_CODE\":\"O\",\n",
+    "                                \"EMAIL_ADDRESS\":\"Email\",\n",
+    "                                \"DOMAIN_NAME\":\"URL\",\n",
+    "                                \"IP_ADDRESS\":\"IPAddress\",\n",
+    "                                \"IBAN_CODE\":\"InternationalBankingAccountNumber\",   \n",
+    "                                \"US_DRIVER_LICENSE\":\"USDriversLicenseNumber\"\n",
+    "                                }\n",
+    "evaluator = Evaluator(model=model)\n",
+    "dataset_ = Evaluator.align_entity_types(\n",
+    "    deepcopy(dataset), entities_mapping=i2b2_entities_to_text_analytics\n",
+    ")\n",
+    "\n",
+    "evaluation_results = evaluator.evaluate_all(dataset_)\n",
+    "results = evaluator.calculate_score(evaluation_results)\n",
+    "\n",
+    "# update params tracking\n",
+    "params = {\"dataset_name\": dataset_name, \"model_name\": model_name}\n",
+    "params.update(model.to_log())\n",
+    "experiment.log_parameters(params)\n",
+    "experiment.log_dataset_hash(dataset)\n",
+    "experiment.log_metrics(results.to_log())\n",
+    "entities, confmatrix = results.to_confusion_matrix()\n",
+    "experiment.log_confusion_matrix(matrix=confmatrix, labels=entities)\n",
+    "\n",
+    "# end experiment\n",
+    "experiment.end()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
we have a new contribution for plotting accuracy metrics. Have you tried it? could be nicer than the plots we had before.